### ISSUE TITLE
Inline links - focus styles

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#427]
**Is your feature request related to a problem? Please describe.**
Focus styles of inline links do not meet UX requirements.

**Describe the solution you'd like**
Inline links should have the following styles in a focussed state:
- background-color: `#0F556C` bluejay
- border: 2px solid `#0F556C` bluejay
- border-bottom: 1px solid `#F5F3F1` oat-light
- color: `#F5F3F1` oat-light

**Additional context**
- This issue may be worked on at the same time as any PR in the following group: #424, #425, #426, #427

**Testing**
- Force focused state
- Tab through links and ensure can tab through to link
- Cypress tests

**Screenshot of designs**
![image](https://user-images.githubusercontent.com/7791726/43959285-24aed20a-9ca6-11e8-991a-7edc8cacaee0.png)

**Comment by nataliesmart** [comment]
Should be as follows:
color: #F5F3F1 oat-light
border-bottom-color: 1px solid #F5F3F1 oat-light
background-color: #0F556C $bluejay
border: 2px solid #0F556C $bluejay
browser default styling e.g. outline

**Comment by sareh** [comment]
Thanks, Natalie!
